Packages/com.unity.inputsystem/ValidationExceptions.json

```diff
@@ -1,4 +1,9 @@
 {
-    "ErrorExceptions": [],
+    "ErrorExceptions": [
```

Comment:
On reflection - we think this should remain out of the exception list. Stefan will do a follow up PR to remove this again.